mono.yaml

```
@@ -0,0 +1,99 @@
+package:
+  name: mono
+  version: 6.14.0
+  epoch: 0
+  description: The mono framework.
+  copyright:
+    - license: Apache-2.0
+    - license: MIT
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  environment:
+    CFLAGS: "-Wno-int-conversion -DMONO_ARCH_NOMAP32BIT"
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bash
+      - build-base
+      - busybox
+      - cmake
+      - gdb
+      - gettext
+      - krb5
+      - libtool
+      - python3
+      - wget
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.winehq.org/mono/mono.git
+      tag: mono-${{package.version}}
+      expected-commit: 3d1eb55c535487d608ef4b57aca330803a6e002d
+      destination: /home/build/src
+
+  - working-directory: /home/build/src
+    pipeline:
+      - runs: |
+          PATH=/usr/lib/mono/bin:$PATH
+          mkdir -p /usr/lib/mono
+          chown -R $(whoami) /usr/lib/mono
+          ./autogen.sh --prefix=/usr/lib/mono
+          make
+          make install
```

```suggestion
     -  uses: autoconf/make
     
     - uses: autoconf/make-install
```

The [`autoconf/make-intall`](https://github.com/chainguard-dev/melange/blob/d0f902b62b439ec416c6f01e558086758ece32f9/pkg/build/pipelines/autoconf/make-install.yaml#L20) pipeline also sets the DESTDIR to the package content and finally does also some cleanup